ten-simple-rules-dockerfiles.Rmd

```diff
@@ -393,21 +392,22 @@ _Listing&nbsp;2_: Example comments.
 
 ## Add metadata as labels
 
-Docker captures useful information in the image metadata automatically, such as the version of Docker used for building the image.
+Docker automatically captures useful information in the image metadata, such as the version of Docker used for building the image.
```

Comment:
Link to example?

Reply:
You mean something like this https://www.ctl.io/developers/blog/post/what-to-inspect-when-youre-inspecting ?

Or maybe we add one to our repo? Unfortunately the official `docker inspect` docs don't show example output, but I've edited the sentence below to say "You can view all labels _and other image metadata_ with the ..."

Reply:
That would work!